README.md

```diff
@@ -16,7 +16,7 @@
 
 # array-smoosh
 
-> [Array.prototype.smoosh and smooshMap](https://github.com/tc39/proposal-flatMap/pull/56) polyfill for all!
+> [Array.prototype.smoosh and smooshMap](https://github.com/tc39/proposal-flatMap/pull/56) ponyfill for all!
```

Comment:
First of all: thank you. :)

I think this is not a ponyfill, because it sets "Array.prototype.{smoosh,smooshMap}" as you may see here https://github.com/nicolaisueper/array-smoosh/blob/master/src/index.js#L57-L58

I described this in the readme 
> No conditional monkey patching, so the web will never break your website 😱

Reply:
Yeah. You’re absolutely right. Going to close this now.